boards/arm/nucleo_wb55rg/nucleo_wb55rg_defconfig

```diff
@@ -3,6 +3,14 @@ CONFIG_SOC_STM32WB55XX=y
 # 32MHz system clock
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=32000000
 
+CONFIG_STM32_LPTIM_TIMER=y
+CONFIG_STM32_LPTIM_CLOCK_LSE=y
+CONFIG_STM32_LPTIM_CLOCK_LSI=n
+# CONFIG_DEBUG=y
+CONFIG_TICKLESS_KERNEL=y
+CONFIG_TIMESLICING=n
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+
```

Comment:
Do not impact default board configuration.
LPTIM conditional configuration could be done in board Kconfig.defconfig, but not here

Reply:
The config for STM32_LPTIM_CLOCK  is now fixed to LSE, 
the config for SYS_CLOCK_TICKS_PER_SEC is fixed 
for all socs 
in the soc/arm/st_stm32/common/Kconfig.defconfig.series